pkg/endpoint/bpf.go

```diff
@@ -640,11 +656,13 @@ func (e *Endpoint) runPreCompilationSteps(regenContext *regenerationContext) (pr
 	// Note that the incoming policy can be the same as the previous policy in cases
 	// where an unnecessary policy computation was skipped. In that case
 	// e.desiredPolicy == e.realizedPolicy also after this call.
-	if err := e.setDesiredPolicy(policyResult, datapathRegenCtxt); err != nil {
+	if err := e.setDesiredPolicy(datapathRegenCtxt.policyResult, datapathRegenCtxt); err != nil {
```

Comment:
I guess there's no need for `policyResult` to be a separate param anymore since it's contained inside `datapathRegenCtxt` which is already being passed. I'm assuming you didn't cut this function down to 1 param due to having to update the signature everywhere else?

Reply:
Changed to one parameter in the last commit now!